editor/editor_file_system.cpp

```diff
@@ -591,47 +640,98 @@ bool EditorFileSystem::_test_for_reimport(const String &p_path, bool p_only_impo
 			return false; // parse error
 		}
 		if (!assign.is_empty()) {
-			if (!p_only_imported_files) {
-				if (assign == "source_md5") {
-					source_md5 = value;
-				} else if (assign == "dest_md5") {
-					dest_md5 = value;
-				}
+			if (assign == "source_md5") {
+				source_md5 = value;
+			} else if (assign == "dest_md5") {
+				dest_md5 = value;
 			}
 		}
 	}
 
-	//imported files are gone, reimport
-	for (const String &E : to_check) {
-		if (!FileAccess::exists(E)) {
+	//check source md5 matching
```

Comment:
```suggestion
	// Check source md5 matching.
```
Same with other comments.

Reply:
Done!

Reply:
You only fixed this one comment ;)

Reply:
You are right! Done!